lxd/images.go

```diff
@@ -130,7 +130,7 @@ func unpack(d *Daemon, file string, path string, sType storageType) error {
 		// Check if we're running out of space
 		if int64(fs.Bfree) < int64(2*fs.Bsize) {
 			if sType == storageTypeLvm {
-				return fmt.Errorf("Unable to unpack image, run out of disk space (consider increasing your pool's volume.size).")
+				return fmt.Errorf("Unable to unpack image, run out of disk space.")
```

Comment:
Oh that's a merge conflict on my end. You seem to have changed it. Before it was reporting a specific message refering to the old storage key.